test/test_torch.py

```diff
@@ -2855,6 +2859,7 @@ def test_dtypes(self):
         all_dtypes = torch.testing.get_all_dtypes()
         do_test_dtypes(self, all_dtypes, torch.strided, torch.device('cpu'))
         if torch.cuda.is_available():
+            all_dtypes.remove(torch.bfloat16)  # Remove once _th_zero_ is enabled on cuda for bfloat16
```

Comment:
as with the unfold test below, it's better to run the actual test and assert it raises an exception so you force the test to get updated.  As it is, we'll probably forget to actually test this.

Reply:
Fixed for all the tests in this file but cant update this test due to a weird behavior in common_utils.py. This test will be updated in the next PR in the stack and CUDA path will work.

Reply:
> cant update this test due to a weird behavior in common_utils.py

OOC, what kind of behavior?

Reply:
In common_utils.py, in do_test_dtypes i would add something like: 
`if dtype == torch.bfloat16 and device == 'cuda:0':` but it will be ignored even when
dtype value is torch.bfloat16 and device is "cuda:0". The problem was with comparing the device variable value to 'cuda:0' - it would never pass.

Same check would work in test_torch.py with no problems.

Reply:
I was not aware that `device` comparison with a string was a supported operation :) What I would expect to work is if you did `device == torch.device('cuda:0')`